src/cdk/overlay/dispatchers/overlay-keyboard-dispatcher.spec.ts

```diff
@@ -178,6 +183,21 @@ describe('OverlayKeyboardDispatcher', () => {
     expect(overlayTwoSpy).not.toHaveBeenCalled();
     expect(overlayOneSpy).toHaveBeenCalled();
   });
+
+  it('should not run change detection if there are no `keydownEvents` observers', () => {
+    spyOn(appRef, 'tick');
```

Comment:
Rather than spying on the `ApplicationRef`, it might be easier to assert that it's outside the `NgZone` using `NgZone.isInAngularZone()`.

Reply:
Umm, could you show an example?

Because I am testing exclusively that `tick()` is not called, instead that something is called inside the zone or outside the zone.

Reply:
My initial thought was that we could bind another listener and check if it's in the zone, but on a second thought, it won't work. Let's leave it as is.